app/controllers/Application.scala

```diff
@@ -145,3 +145,18 @@ object Application extends Controller {
     Redirect(routes.Application.atAGlance("FlatSpec").url)
   }
 }
+
+
+//
+//  This is used for printing the current year for copyright notices
+//
+import java.util.Calendar
+
+object TheYear {
```

Comment:
I think moving this into its own file should be better, may be under artima.helpers package.

Reply:
i am sorry i was thinking this one is under artima-website, just helpers package should do in this case.

Reply:
Thank you very much!

Reply:
@plinlor please make the refactor that Chee Seng suggested. Push another commit to this branch and reassign it back to one of us. Tnx.